src/main/java/com/mongodb/kafka/connect/source/schema/AvroSchema.java

```diff
@@ -145,15 +145,6 @@ public static Schema fromJson(final String jsonSchema) {
     }
   }
 
-  static Schema createSchema(final org.apache.avro.Schema avroSchema) {
-    return createSchema(avroSchema, false, new Context());
-  }
-
-  static Schema createSchema(
-      final org.apache.avro.Schema avroSchema, final boolean isOptional, final Context context) {
-    return createSchema(avroSchema, isOptional, null, context);
-  }
-
```

Comment:
Removed these methods because they were making code less straightforward without adding something useful. If you think they actually were useful, I will roll this change back and make them `private`.